### PR TITLE
Components

### PR DESCRIPTION
**Pull request by KareemWilson**
### In this Branch, I did the following: 

- Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:

## **Books - the default view:**

- [x] Displays the list of books with the Remove button (no functionality yet).

- [x] Create a component called Book for displaying a single book (receive a title and an author as a prop) and reuse it in a component that displays a list of books.

- [x] Display a form for adding a book (no functionality yet).

- [x] Create a separate component for this form (with inputs for a title and an author

## **Categories:**

- [x] Displays a button "Check status" only.

### CODE REVIEW

File: src/components/Book/Book.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/prop-types */
```
Comment (Munsa1) [inline src/components/Book/Book.jsx]: - [ ] Kindly remove disable linters comment as linters help us to write clean and industry standard code

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+);
```
Comment (Munsa1) [inline src/index.js:12]: - [ ] `index.js` should have 2 `<Link>`s in the navbar which leads to the booklist and categories locations
- [ ] It should also have 2 `<Route>`s, one for the BookList component and the other for the Categories component
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @KareemWilson, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._
- [ ] It is a requirement that you have a booklist component that renders a list of books using the `Book` component . Also the booklist component should renders a form by using the CreateNewBook component  or as in your case the `form.jsx` component.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
Hi @KareemWilson , 

Thanks for making the requested change

### Highlights ✨ 

- Code base looks clean 👍 
- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**